domenetjenester/perioder/src/main/java/no/nav/ung/sak/ytelseperioder/YtelsesperiodeDefinisjon.java

```diff
@@ -0,0 +1,22 @@
+package no.nav.ung.sak.ytelseperioder;
+
+import java.time.LocalDate;
+import java.util.Objects;
+
+public record YtelsesperiodeDefinisjon(String definisjon) {
```

Comment:
Det er kanskje tydeligere om vi bruker Month (eller kanskje enda bedre YearMonth) isdf String her, for å tydeliggjøre også gjennom typing hvordan vi periodiserer?

Reply:
No har eg liksom prøvd å lage eit generelt konsept her. Dersom vi bruker YearMonth så faller det litt bort